pkg/controller/controller.go

```diff
@@ -47,7 +47,7 @@ func findNamespace(ns string) string {
 // RegisterInformers creates new informer controllers to watch k8s resources
 func RegisterInformers(c *config.Config) {
 	sendMessage(c, fmt.Sprintf(controllerStartMsg, c.Settings.ClusterName))
-	startTime = time.Now().Local()
+	startTime = time.Now()
```

Comment:
Whats difference between `time.Now()` and `time.Now().Local()` ?

Reply:
Local() sets location with Time